docs/docs/how-tos/use-gpus.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: use-gpus
+title: Using GPUs on Nebari
+description: Provide a complete overview of using GPUs on Nebari, including server setup, environment setup, and validation.
+---
+
+# Using GPUs on Nebari
+## Introduction
+Provide a complete overview of using GPUs on Nebari, including server setup, environment setup, and validation.
+
+## 1. Starting a GPU server
+
+Follow Steps 1 to 3 in the [Authenticate and launch JupyterLab][login-with-keycloak] tutorial. The UI will show a list of profiles (instances, servers, machines).
+
+![Nebari select profile](/img/how-tos/nebari_select_profile.png)
+Your administrator pre-configures these options, as described in [Profile Configuration documentation][profile-configuration].
+
+Select an appropriate profile and click "Start".
+
+### Understanding GPU setup on the server.
+The following steps describe how to get CUDA-related information from the server.
+1. Once your server starts, it will redirect you to a JupyterLab home page.
+2. Click on the **"Terminal"** icon.
+3. Run the command `nvidia-smi`. The top right corner of the command's output should have the highest supported driver.
+    ![nvidia-smi-output](/img/how-tos/nvidia-smi-output.png)
+
+    If you get `nvidia-smi: command not found`. You are most likely on a non-GPU server. Shutdown your server, and start up a GPU-enabled server.
+
+## 2. Creating environments
+
+By default, `conda-store` will build CPU-compatible packages. To build GPU-compatible packages, we the following.
+### Build a GPU-compatible environments
+By default, conda-store will build CPU-compatible packages. To build GPU-compatible packages, we have a couple of options:
+1. **Creating config using PyTorch documentation**:
+
+    You will need to check PyTorch documentation to [select a supported version of CUDA from the support matrix](https://pytorch.org/get-started/locally/) to determine the latest override version for `CONDA_OVERRIDE_CUDA`.
+    Select the following options to get the latest CUDA version:
+    - PyTorch Build = Stable
+    - Your OS          = Linux
+    - Package          = Conda
+    - Language         = Python
+    - Compute Platform = 12.1 (Choose the latest supported platform)
```

Comment:
```suggestion
At the time of writing, 12.1 was the latest supported version. Select the version that matches the `nvidia-smi` output (see above) on your server. 
```

Reply:
My point was more that you actually DONT want to get the latest CUDA from that interface - you want the CUDA version that matches the GPU that you're running (or less).

Reply:
Actually I was also making a different point. :) 

I want to be sure a person who uses a screen reader can get the information if the list is removed.  I also want to help the user understand what choices to make when using the matrix.  For example, is it ok to use LibTorch on Nebari?  Let's tell them to choose linux, conda, and python.

As far as cuda version, the user will want to know which version of cuda is closest to the one given by `nvidia-smi`, which is unknown until running the matrix.  For example, only 12.1 and lower are compatible right now, so if the user tries to install 12.3, it will fail.